dev-tools/mage/gotest_test.go

```diff
@@ -194,47 +194,47 @@ var wantTestAssertOutput = `(?sm:
 === Failed
 === FAIL: dev-tools/mage TestGoTest_Helper_AssertOutput/assert_fails.*
     gotest_test.go:\d+:.*
-        	Error Trace:	gotest_test.go:\d+.*
+        	Error Trace:	.*gotest_test.go:\d+.*
```

Comment:
Updating `testify` to 1.8.0+ changes traces to include the full filepath; the regex here has been updated to do a generic match, see https://github.com/stretchr/testify/releases/tag/v1.8.0